### ISSUE TITLE
Intercept prior has disappeared

### ISSUE THREAD

**Issue by OliJimbo** [paul-buerkner/brms#618]
Hello!

I've just been running some code that I used for prior predictive checks back in October and the posterior_samples() is no longer extracting the prior from the intercept.  This appears to have been working in October, but I have since upgraded.

Is there a work around to access the prior for this parameter?  I've tried sampling it directly but it just returns null ```posterior_samples(full.bm_, pars = "prior_b_Intercept")```

prior_b_Intercept
<img width="251" alt="Screenshot 2019-03-11 at 18 07 27" src="https://user-images.githubusercontent.com/16899600/54147063-3e3c5500-4429-11e9-8a16-e380abc7f3be.png">

**Comment by paul-buerkner** [comment]
This is unfortunate but intentional. I few months back, I discorved a bug when sampling from the Intercept prior if `sample_prior = "yes"`. Since there was no workaround in the current framework, I had to exclude the intercept from the prior sampling alltogether. There are two alternatives:

1) If you want to perform prior predictive checks, simply use `sample_prior = "only"`.
2) If you want to sample from the intercept when `sample_prior = "yes"`, use `... ~ 0 + Intercept + ...` in the model formula.
